### DISCUSSION TITLE
Back/Forward Remaps don't work

### DISCUSSION THREAD

**Discussion by i-zanis** [JetBrains/ideavim#477] [Q&A]
Hello there,

I am trying to use this shortcuts however they do not seem to work. I turned on the ID track C-o says action Id back however it does not function the same way as the mouse button. 
C-i says cannot detect action id. 

noremap <C-o> :action Back<CR>
noremap <C-i> :action Forward<CR>

**Comment by citizenmatt** [comment]
Try using the `<Action>(Back)` syntax, such as `nmap <C-o> <Action>(Back)`. This is a special syntax (like `<Plug>(…)` for plugin commands) that will invoke an action. The `:action Back<CR>` syntax is interpreted as a string of characters, so starts a command line session, types `action Back` and hits return. Because they're invoked differently, there can be different results.

Note that you can't use `noremap` here because we need to remap the `<Action>(Back)` into a command, rather than treat it as a literal string of characters.
